js/pages/incidence-rates/ir-manager.js

```diff
@@ -153,6 +164,23 @@ define([
 			});
 			this.expressionMode = ko.observable('import');
 
+			this.generatedSql = {};
+			this.generatedSql.mssql = ko.observable('');
```

Comment:
Looks like code duplication. Why cannot we do something like:
```
// Place into some app-wide const.js
const dialects = ['mssql', 'oracle', 'postgresql', ...]

// Constructor
dialects.forEach(d => {
  this.generatedSql[d] = ko.observable('');
})

// showSql
const promises = dialects.map(d => {
  return IRAnalysisService.translateSql(result.templateSql, d).then(({data}) => this.generatedSql[d](data.targetSQL));
})
Promise.all(promises).then.....
```

Reply:
done